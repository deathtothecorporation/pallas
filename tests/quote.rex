; comments are
= comments ; totally ignored
| some code; they affect nothing
| some code
| some code

= strings
- 'foo'
- """ foo
- 'doesn''t even'
- "he said ""whoa"", then made the leap"
- """ doesn't even
- """ foo
  """ bar
