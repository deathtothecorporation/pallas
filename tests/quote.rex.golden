= comments
| some code
| some code
| some code

= strings
- 'foo'
-
   """ foo
- 'doesn''t even'
- "he said ""whoa"", then made the leap"
-
   """ doesn't even
-
   """ foo
   """ bar
